source-monday/source_monday/api.py

```diff
@@ -246,11 +231,18 @@ async def fetch_items_changes(
             max_updated_at_in_window = max(max_updated_at_in_window, created_at)
             docs_emitted += 1
             yield deleted_item
+        # TODO(justin): handle other events that might require fetching items (e.g., duplicating a board with items)
+        elif "board_state_change" in activity_log.event:
```

Comment:
The Monday.com API docs do not document any of the Activity Log events. So I have had to discover this through testing and use of their API. This TODO may need to be removed or just left as a note, since right now I am unsure a straight forward way to know a board was duplicated. In testing, when I duplicate a board with all items it shows events that don't mention duplication or anything related. My only idea right now is to maybe key of some `create` type event and determine if a board was just created and then just trying to fetch it's items in case it was a duplicated board+items.